example/scatter-plot-grouping.md

```diff
@@ -0,0 +1,26 @@
+# Scatter Plot Grouping
+
+ With the help of a [Simple X/Y Plot](http://d3plus.org/docs/#Plot), you can group scatter plot by passing the `groupBy` property an array instead of a single key. Based on the keys (and order) you pass it in the array, [ X/Y Plot](http://d3plus.org/docs/#Plot) will group scattered plot.
```

Comment:
> You can group the bubbles in a Scatter Plot by passing the `groupBy` property a nested Array of keys instead of a single key. This enables automatic shape groupings/nestings and click behavior to dive into groups. The default level shown is defined using the `depth` method.

👆 add docs links for `groupBy` and `depth`